src/functionsShellCommandAction.ts

```diff
@@ -28,7 +29,7 @@ export const actionFunction = async (options: any) => {
     debugPort = commandUtils.parseInspectionPort(options);
   }
 
-  const hubClient = new EmulatorHubClient(options.project);
+  const hubClient = new EmulatorHubClient(options.project as string);
```

Comment:
Why do we need a cast here?

Reply:
Previously `options` was `any` so `options.project` was also `any`. Now `options` is `Options` so `options.project` is `unknown`.  We have to assert the type as `string`.  It's not type-safe, but it wasn't before either. The slight improvement here (IMO) is putting the ugly cast out in the open.

I'll change this to use an assertion instead.